diff --git a/bar-functions/dwm_backlight.sh b/bar-functions/dwm_backlight.sh
index c0968d8..185d3db 100755
--- a/bar-functions/dwm_backlight.sh
+++ b/bar-functions/dwm_backlight.sh
@@ -4,10 +4,10 @@
 # Joe Standring <git@joestandring.com>
 # GNU GPLv3
 
-# Dependencies: xbacklight
+# Dependencies: light
 
 dwm_backlight () {
-    printf "%s☀ %.0f%s\n" "$SEP1" "$(xbacklight)" "$SEP2"
+    printf "%s☀ %.0f%s\n" "$SEP1" "$(light)" "$SEP2"
 }
 
 dwm_backlight
diff --git a/bar-functions/dwm_battery.sh b/bar-functions/dwm_battery.sh
index 6321e64..5f4a5d1 100755
--- a/bar-functions/dwm_battery.sh
+++ b/bar-functions/dwm_battery.sh
@@ -6,8 +6,8 @@
 
 dwm_battery () {
     # Change BAT1 to whatever your battery is identified as. Typically BAT0 or BAT1
-    CHARGE=$(cat /sys/class/power_supply/BAT1/capacity)
-    STATUS=$(cat /sys/class/power_supply/BAT1/status)
+    CHARGE=$(cat /sys/class/power_supply/BAT0/capacity)
+    STATUS=$(cat /sys/class/power_supply/BAT0/status)
 
     printf "%s" "$SEP1"
     if [ "$IDENTIFIER" = "unicode" ]; then
diff --git a/dwm_bar.sh b/dwm_bar.sh
index 4c2f8d6..89505f3 100755
--- a/dwm_bar.sh
+++ b/dwm_bar.sh
@@ -24,31 +24,32 @@ export SEP2="]"
 # Import the modules
 #. "$DIR/bar-functions/dwm_alarm.sh"
 #. "$DIR/bar-functions/dwm_alsa.sh"
-#. "$DIR/bar-functions/dwm_backlight.sh"
-#. "$DIR/bar-functions/dwm_battery.sh"
+"$DIR/bar-functions/dwm_backlight.sh"
+"$DIR/bar-functions/dwm_battery.sh"
 #. "$DIR/bar-functions/dwm_ccurse.sh"
-#. "$DIR/bar-functions/dwm_cmus.sh"
+"$DIR/bar-functions/dwm_cmus.sh"
 #. "$DIR/bar-functions/dwm_connman.sh"
 #. "$DIR/bar-functions/dwm_countdown.sh"
 #. "$DIR/bar-functions/dwm_currency.sh"
 #. "$DIR/bar-functions/dwm_date.sh"
 #. "$DIR/bar-functions/dwm_keyboard.sh"
-#. "$DIR/bar-functions/dwm_loadavg.sh"
+"$DIR/bar-functions/dwm_loadavg.sh"
 #. "$DIR/bar-functions/dwm_mail.sh"
-#. "$DIR/bar-functions/dwm_mpc.sh"
-#. "$DIR/bar-functions/dwm_networkmanager.sh"
-#. "$DIR/bar-functions/dwm_pulse.sh"
-#. "$DIR/bar-functions/dwm_resources.sh"
+"$DIR/bar-functions/dwm_mpc.sh"
+"$DIR/bar-functions/dwm_networkmanager.sh"
+"$DIR/bar-functions/dwm_pulse.sh"
+"$DIR/bar-functions/dwm_resources.sh"
 #. "$DIR/bar-functions/dwm_spotify.sh"
 #. "$DIR/bar-functions/dwm_transmission.sh"
 #. "$DIR/bar-functions/dwm_vpn.sh"
 #. "$DIR/bar-functions/dwm_weather.sh"
+"$DIR/bar-functions/dwm_wpa.sh"
 
 parallelize() {
     while true
     do
         printf "Running parallel processes\n"
-        #dwm_networkmanager &
+        dwm_networkmanager &
         #dwm_weather &
         sleep 5
     done
@@ -62,28 +63,29 @@ do
     upperbar=""
     #upperbar="$upperbar$(dwm_alarm)"
     #upperbar="$upperbar$(dwm_alsa)"
-    #upperbar="$upperbar$(dwm_backlight)"
-    #upperbar="$upperbar$(dwm_battery)"
+    upperbar="$upperbar$(dwm_backlight)"
+    upperbar="$upperbar$(dwm_battery)"
     #upperbar="$upperbar$(dwm_ccurse)"
-    #upperbar="$upperbar$(dwm_cmus)"
+    upperbar="$upperbar$(dwm_cmus)"
     #upperbar="$upperbar$(dwm_connman)"
     #upperbar="$upperbar$(dwm_countdown)"
     #upperbar="$upperbar$(dwm_currency)"
     #upperbar="$upperbar$(dwm_date)"
     #upperbar="$upperbar$(dwm_keyboard)"
-    #upperbar="$upperbar$(dwm_loadavg)"
+    upperbar="$upperbar$(dwm_loadavg)"
     #upperbar="$upperbar$(dwm_mail)"
-    #upperbar="$upperbar$(dwm_mpc)"
-    #upperbar="$upperbar$(dwm_pulse)"
-    #upperbar="$upperbar$(dwm_resources)"
+    upperbar="$upperbar$(dwm_mpc)"
+    upperbar="$upperbar$(dwm_pulse)"
+    upperbar="$upperbar$(dwm_resources)"
     #upperbar="$upperbar$(dwm_spotify)"
     #upperbar="$upperbar$(dwm_transmission)"
     #upperbar="$upperbar$(dwm_vpn)"
-    #upperbar="$upperbar${__DWM_BAR_NETWORKMANAGER__}"
+    upperbar="$upperbar$(dwm_wpa)"
+    upperbar="$upperbar${__DWM_BAR_NETWORKMANAGER__}"
     #upperbar="$upperbar${__DWM_BAR_WEATHER__}"
    
     # Append results of each func one by one to the lowerbar string
-    lowerbar=""
+    #lowerbar=""
     
     xsetroot -name "$upperbar"
     # Uncomment the line below to enable the lowerbar 
