diff --git a/config.def.h b/config.def.h
index 93a875a..3b247d7 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,12 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ datetime,  		"%s",            	"%a,%m %d - %T " },
+	{ battery_perc, "| \uf578 %s ", 		"BAT0"},
+	{ uptime,      	"| %s ",        	NULL},
+	{ ram_used,    	"|  %s ",        NULL},
+	{ cpu_perc,    	"|  %s ",   			NULL},
+	{ wifi_essid, 	"| %s -", 				"wlo1"},
+	{ wifi_perc, 		" %s ", 						"wlo1"},
+	{ run_command, 	"| %s ", 					"wpctl get-volume @DEFAULT_AUDIO_SINK@"},
 };
